File: Heat/Models/ProposedOutBoundCall.vb
Namespace Models

    ''' <summary>
    ''' Rappresenta una chiamata per il rinnovo di una manutenzione in scadenza
    ''' </summary>
    Public Class ProposedOutBoundCall

        Property ID As Integer
        ''' <summary>
        ''' L'utente a cui è stata proposta la chiamata.
        ''' </summary>
        ''' <returns></returns>
        Property User As String
        Property PlantID As Integer
        Property Plant As Plant
        Property Contacts As List(Of Contact)

    End Class
End Namespace

File: Heat.ConvertedToC#/Models/ProposedOutBoundCall.cs
using System.Collections.Generic;
namespace Heat.Models
{

    /// <summary>
    /// Rappresenta una chiamata per il rinnovo di una manutenzione in scadenza
    /// </summary>
    public class ProposedOutBoundCall
	{
       
		public int ID { get; set; }
		/// <summary>
		/// L'utente Heat a cui è stata proposta la chiamata.
		/// </summary>
		/// <returns></returns>
		public string User { get; set; }
		public int PlantID { get; set; }
		public Plant Plant { get; set; }
		//public virtual List<Contact> Contacts { get; set; }

	}
}
